src/scene/vcQueryNode.h

```diff
@@ -47,4 +48,21 @@ class vcQueryNode : public vcSceneItem
   vdkQueryFilter *m_pFilter;
 };
 
+struct vdkProjectNode;
+struct vcQueryNodeFilterInput
+{
+  vcQueryNodeFilterShape shape = vcQNFS_None;
```

Comment:
You should try to avoid this, it requires the constructor is called to set this. This will implicitly happen in most cases, but a `udChunkedArray` of this struct won't call the constructor.

Reply:
Deleted.